YonahDiaz-StockExchange/js/company.js

```diff
@@ -0,0 +1,118 @@
+let urlParams = new URLSearchParams(window.location.search);
+let stringUrlParams = urlParams.toString();
+let companySymbol = stringUrlParams.slice(7);
```

Comment:
there is better and easier way to handle url params with URLSearchParams interface. look for it in the MDN docs